### DISCUSSION TITLE
Awesome. Works on RTX 2080

### DISCUSSION THREAD

**Discussion by syllebra** [basujindal/stable-diffusion#136] [Show and tell]
Just  want to send a huge thank you: I can run almost 4 K on RTX 2080. Keep up!

**Comment by prettydeep** [comment]
B/c the SD model was based on 512x512 images, do you notice any change in the quality of content generated at 4K res?

For me (gtx1080), the best quality images are usually generated at 512x512, and then I try to upscale using img2img followed by esrgan.

**Reply by syllebra** [reply]
Yes I do. I don't need to re-upscale after. I used "turbo" option as 2080 RTX have 8Go VRAM, works great!

**Comment by vicento** [comment]
Syllebra What about quality and time needed to render in 4K ¿
